scripts/sizeDiffTable.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+struct SizePair {
+    var oldSize: Int = 0
+    var newSize: Int = 0
+    var delta: Int {
+        return newSize - oldSize
+    }
+}
+var sizeDict: [String: SizePair] = [:]
+
+if CommandLine.arguments.count != 3 {
+    print("usage: swift sizeDiffTable.swift <path to old libFluentUI.a> <path to new libFluentUI.a>")
+} else {
+    let oldPath: String = CommandLine.arguments[1]
+    let newPath: String = CommandLine.arguments[2]
+
+    do {
+        parseArFor(path: oldPath, isOld: true)
+        parseArFor(path: newPath, isOld: false)
+
+        print("| File | Before | After | Delta |")
+        print("|------|--------|-------|-------|")
+
+        let sortedDict = sizeDict.sorted { $0.value.delta > $1.value.delta }
+        for element in sortedDict {
+            let value = element.value
+            let delta = value.delta
+            let numberFormatter = NumberFormatter()
+            numberFormatter.numberStyle = .decimal
+            let beforeString = numberFormatter.string(from: NSNumber(value: value.oldSize)) ?? "0"
+            let afterString = numberFormatter.string(from: NSNumber(value: value.newSize)) ?? "0"
+            let deltaString = numberFormatter.string(from: NSNumber(value: delta)) ?? "0"
+            if delta > 250000 {
```

Comment:
make these define as static const so then we change change these gidelines easily.